adl/release.adl

```diff
@@ -11,6 +12,9 @@ struct ReleaseConfig
     String prestartCommand;
     String startCommand;
     String stopCommand;
+
+    /// Save json context used for cfg templating to file
```

Comment:
Despite the commit, I still don't understand what this string is for.

Reply:
it's an opt-out of being required to use mustache in order to access the final json data.  There is an option to save the json data to a file.  An app could then choose to write it's own config data model and setup matching json contexts and therefore avoid using mustache templates to rearrange json data.

Reply:
That make sense. Perhaps:

```
     Maybe<FilePath> saveCtxJson = "nothing"l
```
would be clearer?